custom/aaa/views.py

```diff
@@ -47,58 +53,73 @@ class ProgramOverviewReport(ReachDashboardView):
 @method_decorator([login_and_domain_required, csrf_exempt], name='dispatch')
 class ProgramOverviewReportAPI(View):
     def post(self, request, *args, **kwargs):
-        # TODO add query to database
         selected_month = int(self.request.POST.get('selectedMonth'))
         selected_year = int(self.request.POST.get('selectedYear'))
         selected_location = self.request.POST.get('selectedLocation')
+        selected_date = date(selected_year, selected_month, 1)
+        prev_month = date(selected_year, selected_month, 1) - relativedelta(months=1)
+
+        location_filters = build_location_filters(selected_location)
+        data = AggVillage.objects.filter(
```

Comment:
@lwyszomi FYI I went ahead and hooked the in the backend for the MoH hierarchy. Feel free to move this around when you add support for MWCD and use AggAwc for those

Reply:
@emord Is there a possibility to get a database dump for these reports? That I could run aggregation scripts on my local machine.